ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParsePayloadTest.java

```diff
@@ -26,7 +27,7 @@
 import org.junit.Rule;
 import org.junit.Test;
 
-public class ParsePayloadTest {
+public class ParsePayloadTest extends TestWithDeterministicJson {
```

Comment:
This is is required to keep the test deterministic. Either bumping jackson version or excluding new fields from serialized JSON caused field order in test output to change.

Following changes in this file are reordering fields in alphabetical order.